__tests__/app.test.js

```diff
@@ -38,6 +38,46 @@ describe("/api/topics", () => {
       });
     });
 });
+
+describe("/api/articles/:article_id", () => {
+  describe("GET", () => {
+    test("200: responds with a single article and contains all the relevant information for it", () => {
+      return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body;
+          expect(article).toEqual({
```

Comment:
We should use a more dynamic way to check, so objectContaining would be an option, have a look at this link for some examples https://www.emgoto.com/jest-partial-match/ 

The reason this test is not ideal is because if we were to add an extra property to the articles in the future than this test would fail 🙂